style.css

```diff
@@ -94,94 +113,367 @@ ul {
+  align-items: center;
+  text-align: center;
+  color: white;
+}
+
+.detail {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #ebebff;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+}
+
+.input {
+  padding: 12px;
+  background: white;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+}
+
+.submited {
+  align-self: flex-start;
+  background: white;
+  border: 1px solid #7f8cff;
+  border-radius: 8px;
+}
+
+.submited:hover {
+  background: #6070ff;
+  box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
+  border-radius: 8px;
+}
+
+.submited:active {
+  background: #2230d2;
+  border-radius: 8px;
+}
+
+.submited:disabled {
+  border: 1px solid #c1c7c0;
+  border-radius: 8px;
+}
+
+@media screen and (min-width: 768px) {
+  #header {
+    padding: 1rem 143px;
   }
 
-  .hamburger {
+  .Menu {
     display: flex;
-    justify-content: space-between;
-    padding: 1rem;
+  }
+
+  .Menuicon {
+    display: none;
   }
 
   #headline_section {
-    background-color: white;
+    padding: 0;
+    height: 80vh;
     display: flex;
-    flex-direction: column;
-    align-items: flex-start;
-    padding: 114px 24px;
-    gap: 12px;
-    background-image: url(./img/imagee.png);
+    align-items: center;
+    justify-content: center;
+    background-image: none;
+    background-image: url(./img/Headerbg.svg);
+    background-position: bottom;
+    background-size: contain;
     background-repeat: no-repeat;
-    background-size: cover;
-    border-radius: 0 0 0 5rem;
   }
 
-  .headline {
+  .bg {
     display: flex;
     flex-direction: column;
-    align-items: flex-start;
-    gap: 2rem;
-    padding: 1rem;
+    align-items: center;
+    justify-content: center;
+    width: 98%;
+    height: 90%;
   }
 
-  #lts_connect {
-    color: var(--primaryColor);
+  .headline {
+    width: 60%;
+    padding: 1rem;
+    gap: 1rem;
   }
 
-  .contact {
-    list-style: none;
-    display: flex;
-    gap: 1rem;
+  .heading {
+    font-size: 48px;
+    line-height: 60px;
+    width: 50%;
   }
 
-  .icon {
-    width: 24px;
-    height: 24px;
+  .description {
+    font-size: 20px;
+    line-height: 28px;
+    width: 80%;
   }
 
   #works {
-    display: grid;
-    align-content: center;
-    padding: 114px 24px;
-    grid-gap: 88px;
+    justify-content: center;
   }
 
-  .page1 {
-    display: grid;
-    align-content: center;
-    padding: 16px;
+  .pages {
+    grid-auto-flow: column;
+    padding: 24px;
     grid-gap: 12px;
-    background: white;
-    border: 1px solid #def1e6;
-    border-radius: 16px;
   }
 
-  .snapshoot_portpolio {
-    width: 295px;
-    height: 220px;
-    background: url(./images/SnapshootPortfolio.svg);
-    border-radius: 8px;
+  .card_image {
+    height: 100%;
+    width: auto;
+    padding-right: 1rem;
   }
 
   .detail1 {
     display: grid;
     align-content: start;
     grid-gap: 12px;
+    height: 465px;
     width: 100%;
   }
```

Comment:
- [ ] Ouch! it seems like the text content in your design of the project cards is being overlayed in the project's image and this is causing a bad UI, as a frontend developer you have to make sure your design looks polished and professional because at any time some of your future employers might take a look at what you're building. ✨
- Your design:
  ![image](https://github.com/medijdir/portfolio_project/assets/104968236/08a7268d-dc9d-4848-b6b9-4c5f3a2ac42a)
- Figma design:
  ![image](https://github.com/medijdir/portfolio_project/assets/104968236/5ab30b58-a0b3-42cd-9b26-693b07717fa4)
- Part of the problem I've identified is the fixed height you gave at line `476` maybe you can consider removing it and adjusting the width of the image as it might get affected. 😉